beetleblocks/objects.js

```diff
@@ -1340,20 +1340,24 @@ StageMorph.prototype.initCamera = function () {
 
         myself.camera.fitScene = function () {
 
-            var boundingBox = new THREE.Box3().setFromObject(myself.myObjects),
-                boundingSphere = boundingBox.getBoundingSphere(),
-                center = boundingSphere.center,
-                distance = boundingSphere.radius;
+          this.reset();
 
-            this.reset();
+          var boundingBox = new THREE.Box3().setFromObject(myself.myObjects),
+              boundingSphere = boundingBox.getBoundingSphere(),
+              center = boundingSphere.center,
+              distance = boundingSphere.radius;
+              console.log(boundingBox)
```

Comment:
needs to be removed

Reply:
Shouldn't be included in the pull request, but now I changed the rest to (hopefully) fix 142.